cypress/e2e/tests/pages/manager/cluster-manager.spec.ts

```diff
@@ -169,6 +176,34 @@ describe('Cluster Manager', { testIsolation: 'off', tags: ['@manager', '@adminUs
         });
 
         detailRKE2ClusterPage.waitForPage(undefined, 'registration');
+
+        cy.get('.checkbox-label').contains('Insecure:').click();
```

Comment:
this should go in a po file. we don't have one for the `registration` tab afaik, so that will need to be created and then this po added to it. see `cypress/e2e/po/detail/provisioning.cattle.io.cluster/cluster-detail-snapshots.po.ts` as an example